lib/__tests__/test-utils.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import * as enzyme from 'enzyme';
+import dateFnsUtils from '../src/utils/date-fns-utils';
+import momentUtils from '../src/utils/moment-utils';
+
+export const utilsToUse = process.env.UTILS === 'moment'
+  ? momentUtils
+  : dateFnsUtils;
+
+const getComponentWithUtils = Component => React.cloneElement(Component, { utils: utilsToUse });
```

Comment:
why not using `withUtils` here

Reply:
Because then shallow rendering would render just a HOC, instead of testing component

Reply:
You can use enzyme `dive` method or use `createShallow({ dive: true })` from` MUI test utils